fga-eps-mds/2020.1-stay-safe-user-service
# Protection improvements

## Issue 
Resolves #38 


## Description
This PR makes changes so that it is not possible for a user to edit or delete another user's ratings and occurrences. It also omits the user's password in gets.

## Checklist:



## How to validate?
1. Run the tests: ` docker-compose run api pytest`
2. Try to edit or delete another user’s occurrence or rating and fail
3. Make a user get and see that the password is not coming
